cli/getConfig.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-console */
+export interface CollectionDefinition {
+  base?: string
+  packageName?: string
+  pattern: string
+  name: string
+}
+
+export interface DocsConfig {
+  content: CollectionDefinition[]
+}
+
+export async function getConfig(fileLocation: string): Promise<DocsConfig | undefined> {
+  try {
+    const { config } = await import(fileLocation)
+    return config as DocsConfig
+  } catch (e: any) {
+    if (['ERR_MODULE_NOT_FOUND', 'MODULE_NOT_FOUND'].includes(e.code)) {
+      console.error(
+        'pf-docs.config.js not found, have you created it at the root of your package?',
+      )
+      return
```

Comment:
Did we want to error out or still continue? Wondering if it should be 'ERROR: pf-dos.config.js not found ...'

Reply:
Issue #20 was created for this.